README.md

```diff
@@ -109,11 +109,49 @@ Upon receipt of a transactions webhook a call will be made to Plaid's transactio
 
 ### Testing OAuth
 
-A redirect_uri parameter is included in the linkTokenCreate call and set in this sample app to the PLAID_SANDBOX_REDIRECT_URI you have set in the .env file (`http://localhost:3001/oauth-link`). This is the page that the user will be redirected to upon completion of the OAuth flow at their OAuth institution. When running in Production or Development, you will need to use an `https://` redirect URI, but a localhost http URI will work for Sandbox.
+A redirect_uri parameter is included in the linkTokenCreate call and set in this sample app to the PLAID_SANDBOX_REDIRECT_URI you have set in the .env file (`http://localhost:3001/oauth-link`). This is the page that the user will be redirected to upon completion of the OAuth flow at their OAuth institution. You will also need to configure `http://localhost:3001/oauth-link` as an allowed redirect URI for your client ID through the [Plaid developer dashboard](https://dashboard.plaid.com/team/api).
 
-You will also need to configure `http://localhost:3001/oauth-link` as an allowed redirect URI for your client ID through the [Plaid developer dashboard](https://dashboard.plaid.com/team/api).
+To test the OAuth flow in sandbox, choose 'Playtypus OAuth Bank' from the list of financial institutions in Plaid Link.
 
-To test the OAuth flow, choose 'Playtypus OAuth Bank' from the list of financial instutions in Plaid Link.
+#### Instructions for using https with localhost
+
+If you want to test OAuth in development, you need to use https and set `PLAID_REDIRECT_URI=https://localhost:3001/` in `.env`. In order to run your localhost on https, you will need to create a self-signed certificate and add it to the client root folder. You can use the following instructions to do this. However, please note that this is recommended for local development environments only and not for production environments.
```

Comment:
i thought the redirect uri was https://localhost:3001/oauth-link?

Reply:
oops...you are right!